builder/dockerfile/copy.go

```diff
@@ -56,6 +56,7 @@ type copyInstruction struct {
 	cmdName                 string
 	infos                   []copyInfo
 	dest                    string
+	flChown                 string
```

Comment:
yes, used commonly in the codebase; although in this struct the flag value is being stored, so toss up whether I should be using the `fl` prefix at this point :)